__tests__/services/playlistManagementService.test.js

```diff
@@ -316,4 +322,43 @@ describe("orderPlaylist triggering conditions", () => {
     expect(mockManagedPlaylist.remove).toHaveBeenCalledTimes(1);
     expect(setInterval).toHaveBeenCalledTimes(0);
   });
+
+  it("The function orderPlaylist should be called when no other orderPlaylist  is running", async () => {
```

Comment:
```suggestion
  it("The function orderPlaylist should be called when no other orderPlaylist  is running for the same playlist", async () => {
```
Assuming refreshToken will not be used. If Refresh Token is really necessary, "same playlist and refreshId"